reference/content.yaml

```diff
@@ -0,0 +1,381 @@
+openapi: 3.1.0
+info:
+  title: Content
+  description: Search our non-catalogue content
```

Comment:
Don't think it's necessarily this simple - I changed it from `editorial` because I felt that `articles` was editorial content and the other (yet-to-exist) endpoints were arguably not. I copied the `non-catalogue` terminology from the original RFC https://github.com/wellcomecollection/docs/tree/main/rfcs/055-prismic-api

Reply:
Ah, I should have been clearer – let's be internally consistent elsewhere in the PR. It says "editorial content", should this say "non-catalogue" instead? https://github.com/wellcomecollection/developers.wellcomecollection.org/pull/22/files#diff-0b5d05c912a16dd4e7dbbf126fde7b74ab4487c95edb1f31ab8aed845e7ca7e0R16